echopype/convert/parse_azfp.py

```diff
@@ -349,6 +349,26 @@ def _test_valid_params(params):
             if isinstance(val, list):
                 self.parameters[key] = np.asarray(val)
 
+        # Get frequency values
+        freq_old = list(self.unpacked_data["frequency"])
+
+        # Sort the frequencies in ascending order
+        freq_new = freq_old[:]
+        freq_new.sort(reverse=False)
+
+        # Obtain sorted frequency indices
+        self.freq_ind_sorted = [freq_new.index(ch) for ch in freq_old]
+
+        # Obtain sorted frequencies
+        self.freq_sorted = self.unpacked_data["frequency"][self.freq_ind_sorted] * 1000.0
```

Comment:
(I know this is not your code, but since we're here...)
Seems [xr.dataarray.argsort](https://docs.xarray.dev/en/latest/generated/xarray.DataArray.argsort.html) can simplify this quite a bit?